upgrade.md

```diff
@@ -471,3 +480,42 @@ Then, in your application's `config/sanctum.php` configuration file, you should
         'encrypt_cookies' => Illuminate\Cookie\Middleware\EncryptCookies::class,
         'validate_csrf_token' => Illuminate\Foundation\Http\Middleware\ValidateCsrfToken::class,
     ],
+
+<a name="telescope"></a>
+### Telescope
+
+<a name="updating-telescope"></a>
+#### Updating Telescope
+
+**Likelihood Of Impact: High**
+
+Laravel 11 no longer supports Laravel Telescope 4.x. Therefore, you should update your application's Laravel Telescope dependency to `^5.0` in your `composer.json` file.
+
+Telescope 5.0 no longer automatically loads migrations from its own migrations directory. Instead, you should run the following command to publish Telescope's migrations to your application:
+
+```shell
+php artisan vendor:publish --tag=telescope-migrations
+```
+
+<a name="passport"></a>
+### Passport
+
+<a name="updating-telescope"></a>
+#### Updating Passport
+
+**Likelihood Of Impact: High**
+
+Laravel 11 no longer supports Laravel Passport 11.x. Therefore, you should update your application's Laravel Passport dependency to `^12.0` in your `composer.json` file.
+
+Passport 12.0 no longer automatically loads migrations from its own migrations directory. Instead, you should run the following command to publish Passport's migrations to your application:
+
+```shell
+php artisan vendor:publish --tag=passport-migrations
+```
+
+In addition, the password grant type is disabled by default. You may enable it by calling the `enablePasswordGrant` method in the `boot` method of your application's `AppServiceProvider`:
+
+    public function boot(): void
+    {
+        Passport::enablePasswordGrant();
+    }
```

Comment:
I don't think we need these extra sections on sanctum, passport and telescope. We should direct people to these package's upgrade guide's instead. Doing the above will only duplicate these upgrade steps in two different places. By just pointing people to the package upgrade guides we can add more steps to those guides in the future and don't need to update two different places.